overlord/assertstate/assertstate.go

```diff
@@ -336,7 +344,8 @@ func delayedCrossMgrInit() {
 
 // AutoRefreshAssertions tries to refresh all assertions
 func AutoRefreshAssertions(s *state.State, userID int) error {
-	if err := RefreshSnapDeclarations(s, userID); err != nil {
+	opts := &RefreshSnapDeclarationsOptions{IsAutoRefresh: true}
+	if err := RefreshSnapDeclarations(s, userID, opts); err != nil {
 		return err
 	}
 	return RefreshValidationSetAssertions(s, userID)
```

Comment:
this should also take opts basically

Reply:
Should I rename the options to RefreshAssertionsOptions since it's used in both kinds of assertions (snap declarations and validation sets)? Or is it better to have two different options for each type of call?